js/src/annotations/osd-canvas-renderer.js

```diff
@@ -269,9 +269,19 @@
       });
 
     },
-    
+      hideWindowAnnotations: function() {
```

Comment:
When hiding annotations - in case an annotations was removed, the OSD moved or someone started editing an annotation - I want to make sure I only close the annotations on the current window. This is not an issue when you have no 'sticky' annotations, but when you do, you want to make sure you only close the ones belonging to the windows where the event happened.